<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CControls</class>
 <widget class="QWidget" name="CControls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>938</width>
    <height>267</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="_btnCreate10Circles">
         <property name="toolTip">
          <string>Click to create 10 random circles</string>
         </property>
         <property name="text">
          <string>Create 10 circles</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_btnSelectAll">
         <property name="text">
          <string>Select all</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_btnClearSelection">
         <property name="text">
          <string>Clear selection</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_btnRemoveSelected">
         <property name="text">
          <string>Remove selected</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="_btnReset">
         <property name="text">
          <string>Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Fill Color</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <layout class="QGridLayout" name="gridLayout">
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Blue</string>
              </property>
              <property name="buddy">
               <cstring>_bFillColorSlider</cstring>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSlider" name="_bFillColorSlider">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Opacity</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSlider" name="_gFillColorSlider">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>Red</string>
              </property>
              <property name="buddy">
               <cstring>_rFillColorSlider</cstring>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>Green</string>
              </property>
              <property name="buddy">
               <cstring>_gFillColorSlider</cstring>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSlider" name="_rFillColorSlider">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
              <property name="invertedControls">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QSlider" name="_aFillColorSlider">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="minimum">
               <number>32</number>
              </property>
              <property name="maximum">
               <number>255</number>
              </property>
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Radius</string>
           </property>
           <property name="buddy">
            <cstring>_radiusSlider</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSlider" name="_radiusSlider">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
